src/test/erasure-code/test-erasure-code.sh

```diff
@@ -30,16 +30,10 @@ function run() {
 
     setup $dir || return 1
     run_mon $dir a || return 1
-    # check that erasure code plugins are preloaded
-    CEPH_ARGS='' ceph --admin-daemon $dir/ceph-mon.a.asok log flush || return 1
-    grep 'load: jerasure.*lrc' $dir/mon.a.log || return 1
```

Comment:
The reason why plugins must be preloaded is best explained in 9b802701f78288ba4f706c65b853415c69002d27 comments. Does that make sense or has this become irrelevant ?